bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogManagerForEntryLogPerLedger.java

```diff
@@ -554,15 +555,15 @@ public File getDirForNextEntryLog(List<File> writableLedgerDirs) {
     public void close() throws IOException {
         Set<BufferedLogChannelWithDirInfo> copyOfCurrentLogsWithDirInfo = getCopyOfCurrentLogs();
         for (BufferedLogChannelWithDirInfo currentLogWithDirInfo : copyOfCurrentLogsWithDirInfo) {
-            EntryLogger.closeFileChannel(currentLogWithDirInfo.getLogChannel());
+            IOUtils.close(log, currentLogWithDirInfo.getLogChannel());
```

Comment:
any reason why "IOUtils.close" is called instead of "currentLogWithDirInfo.getLogChannel().close()"

Reply:
if null check is what you are looking for then probably doing null check is better instead of calling IOUtils, since it suppresses log message.

Also for the sake of consistency do the same thing as what you are doing in EntryLogManagerForSingleEntryLog 
https://github.com/apache/bookkeeper/pull/1414/files#diff-64c79da08c688a4e03da592ab424d871R215

Reply:
Sure